tests/integration/security/fuzz/fuzz_test.go

```diff
@@ -45,44 +49,138 @@ spec:
 `
 )
 
+var (
+	// Known unsupported path parameter ("/bla;foo") normalization for Tomcat.
+	dotdotPwnIgnoreTomcat = []string{
+		"/../private/secret.html;index.html <- VULNERABLE!",
+		"/../private/secret.html;index.htm <- VULNERABLE!",
+		"/..%5Cprivate%5Csecret.html;index.html <- VULNERABLE!",
+		"/..%5Cprivate%5Csecret.html;index.htm <- VULNERABLE!",
```

Comment:
same question as below - the `index.html` part doesn't matter, right? It could be `index.foobar` and still bypass. So aren't the infinite possible bypasses?

Reply:
yes, there are infinite possible bypasses but the tool will only test a few number of them.